cloud/pkg/cloudhub/channelq/channelq.go

```diff
@@ -63,7 +65,10 @@ func (q *ChannelMessageQueue) DispatchMessage() {
 			klog.Warning("node id is not found in the message")
 			continue
 		}
-
+		if msg.Router.Operation == application.ApplicationResp {
```

Comment:
In isListResource, we also have the judgment logic for the response, do we need to create a new one here?

Reply:
@fisherxu get it, will remove these codes, and replace message.Operation with constant beehiveModel.ResponseOperation. 

beside, I think isListResource and addListResource is confusing, In fact they are down channel for message that do not need realiable transmission. maybe need to rename them later.

Reply:
Agreed, for the first version of design, haven't involved the unrealiable transmission, which is really needed, let's rename it later.